packages/design-system/src/components/button/stories/index.js

```diff
@@ -35,6 +34,23 @@ import { DarkThemeProvider } from '../../../storybookUtils/darkThemeProvider';
 
 export default {
   title: 'DesignSystem/Components/Button',
+  // TODO: fix select dropdown
+  argTypes: {
+    type: {
+      options: Object.values(BUTTON_TYPES),
+    },
+    variant: {
+      options: Object.values(BUTTON_VARIANTS),
+    },
+    size: {
+      options: Object.values(BUTTON_SIZES),
+    },
+  },
```

Comment:
Another example of wanting to use the `select` control from Storybook, but haven't figured how what I am doing wrong.